packages/contracts/contracts/Passport.sol

```diff
@@ -8,34 +8,36 @@ import "@openzeppelin/contracts/access/Ownable.sol";
  * The Passport contract does this and that...
  */
 contract Passport is ERC721, Ownable {
-  uint public passportId = 0;
   address payable public _owner;
   mapping(uint256 => bool) public sold;
-  mapping(uint256 => uint256) public price;
+  uint256 public price;
   event Purchase(address owner, uint256 price, uint256 id, string uri);
-  constructor() ERC721("YOUR TOKEN", "TOKEN") {
+  constructor(string memory name_, string memory symbol_, uint256[] memory tokenIds, uint256 _price) ERC721(name_, symbol_) {
     _owner = payable(msg.sender);
+    uint quantity = tokenIds.length;
+    price = _price;
+    for (uint i = 0; i < quantity; i++) {
+      mint(tokenIds[i]);
+    }
```

Comment:
I think in a future iteration sees us actually using a ERC1155 instead of ERC721, with its batchMint operation